internal/kuberesource/parts.go

```diff
@@ -301,5 +301,12 @@ func ServiceMeshProxy() *applycorev1.ContainerApplyConfiguration {
 			WithPrivileged(true).
 			AddCapabilities("NET_ADMIN").
 			SecurityContextApplyConfiguration,
+		).
+		WithStartupProbe(Probe().
+			WithInitialDelaySeconds(1).
+			WithPeriodSeconds(5).
```

Comment:
I think this could be smaller, say 1s. I measured a couple of container startups and it varied between 0.5s to 0.9s to get to `Invalid path: /tls-config/mesh-ca.pem`, so I'd guess the entire thing should be done in 2s most. But the threshold should be higher accordingly.

Reply:
If we are probing for the Envoy endpoint, we need to wait until the Envoy proxy has started. Looking at the logs, that takes about 4s to 4.3s for me.